build/kube-dns/Makefile

```diff
@@ -0,0 +1,66 @@
+# Copyright 2016 The Kubernetes Authors All rights reserved.
```

Comment:
Maybe add a small readme like hyperkube's.
If you want me to do it, I may create a follow up (it's not so important)